cmd/exporter/exporter.go

```diff
@@ -102,8 +102,16 @@ func setupLogger(level string, output string, logtype string) *slog.Logger {
 func runServer(ctx context.Context, cfg *config.Config, csp provider.Provider, log *slog.Logger) error {
 	mux := http.NewServeMux()
 
-	mux.HandleFunc("/", web.HomePageHandler(cfg.Server.Path)) // landing page
-	registryHandler, err := createPromRegistryHandler(csp)    // prom metrics handler
+	mux.HandleFunc("/", web.HomePageHandler(cfg.Server.Path))               // landing page
+	mux.HandleFunc("/ready", func(w http.ResponseWriter, r *http.Request) { // readiness probe
```

Comment:
Let's align with k8s and use `readyz`: https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health

Reply:
updated